src/Microsoft.Framework.PackageManager/Utils/FrameworkSelectionHelper.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Microsoft.Framework.Runtime.Helpers;
+
+namespace Microsoft.Framework.PackageManager.Utils
+{
+    public static class FrameworkSelectionHelper
+    {
+        public static IEnumerable<FrameworkName> SelectFrameworks(Runtime.Project project,
```

Comment:
When there are no matches, return an empty collection, not null. It makes the consumer code simpler because instead of having to write something like:

```
var result = SelectFrameworks(....);
if (result != null)
{
     foreach(var fx in result) { ... }
}
```

You can write the for loop without the null check.